tests/node/instrumentation-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var path = require('path');
+var distPath = path.join(__dirname, '../../dist');
+var emberPath = path.join(distPath, 'ember.debug');
+
+QUnit.module('instrumentation', function() {
+  QUnit.test('it works in FastBoot environment', function(assert) {
+    var _originalWindow = global.window;
+
+    global.window = {}; // mock window without `performance` property
+    var Ember = require(emberPath);
+
+    var result = Ember.instrument('render', {}, function() {
+      return 'hello';
+    });
+
+    assert.equal(result, 'hello', 'called block');
+
+    global.window = _originalWindow;
+    delete global.Ember;
```

Comment:
Can you move the cleanup to hooks.afterEach? As written this cleanup would not happen if an earlier statement errors...